src/bci_build/package.py

```diff
@@ -1369,6 +1369,31 @@ def _get_openjdk_kwargs(
     )
 ]
 
+PROMETHEUS_PACKAGE_NAME = "golang-github-prometheus-prometheus"
+PROMETHEUS_CONTAINERS = [
+    ApplicationStackContainer(
+        package_name="prometheus-image",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        name="prometheus",
+        pretty_name="Prometheus",
+        package_list=[PROMETHEUS_PACKAGE_NAME],
+        version="%%prometheus_version%%",
```

Comment:
Hm, this is in hindsight not ideal, because now the container name is very weird via the cli. Could you maybe change the version to `2` and add the actual version into the `additional_versions` list?

Reply:
Could you elaborate on this? The container image name is _prometheus_ and does not include version number, right?  
99.9% of users have Prometheus 2.x and adding version 2 to the image name would be confusing, I guess.  
Ideally, I'd like to keep the same image name and version tags as the [upstream images](https://hub.docker.com/r/prom/prometheus/tags).

Reply:
I wanted to keep the "main" version something static, because the dockerfile generator would allow you to write the package to obs via `$name-$version-$os_version`, which would result in the very ugly `Prometheus-%%prometheus_version%%-sp4` name. However, this is a purely non-technical and rather silly limitation, which is why I have created #33.
Could you please rebase on main and add the parameter `version_in_uid=False` to the constructor of `ApplicationStackContainer`?

Reply:
Done